src/javascript/_common/base/currency_base.js

```diff
@@ -79,11 +79,24 @@ const CryptoConfig = (() => {
 
 const getMinWithdrawal = currency => (isCryptocurrency(currency) ? (getPropertyValue(CryptoConfig.get(), [currency, 'min_withdrawal']) || 0.002) : 1);
 
-// returns in a string format, e.g. '0.00000001'
-const getMinTransfer = currency => {
-    const min_transfer = getPropertyValue(currencies_config, [currency, 'transfer_between_accounts', 'limits', 'min']) || getMinWithdrawal(currency);
+/**
+ * Returns the transfer limits for the account.
+ * @param currency
+ * @param which
+ * @returns {min: numeric, ?max: numeric}
+ */
+const getTransferLimits = (currency, which) => {
+    const transfer_limits = getPropertyValue(currencies_config, [currency, 'transfer_between_accounts', 'limits']) || getMinWithdrawal(currency);
     const decimals     = getDecimalPlaces(currency);
-    return min_transfer.toFixed(decimals); // we need toFixed() so that it doesn't display in scientific notation, e.g. 1e-8 for currencies with 8 decimal places
+
+    switch (which) {
+        case 'max':
+            return transfer_limits.max.toFixed(decimals) || undefined;
```

Comment:
in case that we fall back to `getMinWithdrawal(currency)` for `transfer_limits` variable, we won't have `transfer_limits.max` so will `transfer_limits.max.toFixed(decimals)` throw an error?

Reply:
same for below cases, getMinWithdrawal doesn't return an object with `min` key, so it's better to handle that

Reply:
Good point. will do.